rest-server/server/src/main/resources/reference.conf

```diff
@@ -37,3 +37,7 @@ spark {
   eventLog.dir = "file:///tmp"
   job-server-url = ${SPARK_JOB_SERVER_URL}
 }
+
+jobs {
+  expire-after: ${JOBS_EXPIRE_AFTER}
```

Comment:
I thought we agreed to NOT us akka config and ConfigFactory. This is not DRY and has cause MANY bug when we duplicate looking in the env in harness-env and in akka dot conf files.

Are we not on the same page here?